autorest/codegen/models/base_builder.py

```diff
@@ -50,14 +52,23 @@ def __init__(
             overloads or []
         )
         self._summary: str = yaml_data.get("summary", "")
-        # for operations where we don't know what to do, we mark them as abstract so users implement
-        # in patch.py
-        self.abstract = abstract
         self.want_tracing = want_tracing
         self.group_name: str = yaml_data["groupName"]
         self.is_overload: bool = yaml_data["isOverload"]
         self.api_versions: List[str] = yaml_data["apiVersions"]
 
+        if code_model.options["version_tolerant"] and yaml_data.get("abstract"):
+            _LOGGER.warning(
+                'Not going to generate operation "%s" because it we are unable to generate this '
```

Comment:
nit: "because it" --> "because"?

Reply:
thanks! my sentence really didn't make much sense 🤣